### DISCUSSION TITLE
Weather Station addon

### DISCUSSION THREAD

**Discussion by shambahri** [nakhonthai/ESP32APRS_T-TWR#19] [Ideas]
Hi Nakhonthai,

Excellent apps for APRS for ESP32 TWR+ , i have been using your app for few month and i am very happy. Any chance you can add a weather station function to accommodate a simple temperature and humidity sensor for a start. Love to see this function for my station. Thank you again for a great applications.

Kind regards,
Shams (M0DGZ)

**Comment by nakhonthai** [comment]
I am developing this aspect in an ESP32IGate project.
Coming soon.
